msteams-platform/sbs-calling-and-meeting.yml

```diff
@@ -155,7 +155,7 @@ items:
     1. Select your **Resource group** from the dropdown list.
 
         To create a new resource group, select **Create new**, enter resource name, select **OK**, and select required location from **New resource group location** dropdown list.
-
+        
```

Comment:
Not sure but it seems that the article doesn't have an edit except inserting this space. Check if this update was required. Remove, if not required.

Reply:
removed the extra space